lib/tictactoe.rb

```diff
@@ -1,4 +1,6 @@
 require_relative './cli.rb'
+require_relative './content.rb'
+require_relative './game.rb'
 require 'pry'
```

Comment:
Pry shouldn't be required

Reply:
commit 1876366afbfb624b2a53a59ec620ecbd0dacb427

Reply:
commit 61ba9858a6b50abc5ed9c5e92ac21905d8aa9657